### PR TITLE
Te msphere freespace

### PR DESCRIPTION
**Pull request by dccowan**
Time-Domain response of sphere in free-space section created

### CODE REVIEW

File: content/physical_properties/magnetic_permeability/index.rst [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
 **Contents:**
 
  .. toctree::
     :maxdepth: 2
 
-    magnetic_permeability_lab_setup_measurements
+    magnetic_permeability_definition
     magnetic_permeability_units_values
     magnetic_permeability_factors
-    magnetic_permeability_mathematical_relationships
```
Comment (lheagy) [inline content/physical_properties/magnetic_permeability/index.rst]: this delete is causing this branch to fail. Could you put it back?

File: content/maxwell4_tdem/sphere_freespace_TEM.rst [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+Response from a Conducting and Permeable Sphere in Free-Space
+-------------------------------------------------------------
+
+Introduction
+============
+
+Time-domain responses from ore bodies and unexploded ordnance items are of great practical importance to geophysicists.
+On a fundamental level, these responses can be understood by considering the excitation of a conductive and magnetically permeable sphere.
+Wait (1951) was able to characterize a sphere's frequency-domain excitation in terms of an induced dipole moment.
+Here, expressions derived by Wait (1951) are transformed to the time-domain according to Wait and Spies (1969).
```
Comment (thast) [inline content/maxwell4_tdem/sphere_freespace_TEM.rst]: link to your work in FDEM

Comment (thast) [inline content/maxwell4_tdem/sphere_freespace_TEM.rst]: add the article in EM/content/reference.bib to use cite command

File: content/maxwell4_tdem/sphere_freespace_TEM.rst [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+Obtaining the Time-Domain Response of a Conducting and Permeable Sphere
+=======================================================================
+
+.. figure:: ./images/figGeometrySphere.png
+	:align: right
+        :scale: 40%
+        :name: GeometrySphere
+
+
+Consider a conductive and permeable sphere in a resistive media (Figure reference).
```
Comment (thast) [inline content/maxwell4_tdem/sphere_freespace_TEM.rst]: use :numref: command for figure reference

**Comment by lheagy** [comment]
are there still things that need to be updated changed on this pr? Otherwise, @dccowan : could you merge master into this branch (let me know if you need a hand with this!) and then we can get this merged into master!

**Comment by dccowan** [comment]
Hi Lindsey,

The TEM sphere freespace section has been put on the back burner for a
little while. The FEM sphere case was edited but Doug has not provided
final approval. I have a feeling that he will want to re-organize the FEM
sphere example to fit the new website format. In short, the TEM sphere case
is waiting until the FEM sphere case is finished. Make sense?

Devin

**Comment by lheagy** [comment]
yep! thanks for the update

**Comment by lheagy** [comment]
@dougoldenburg : the link to the FEM sphere is here http://em.geosci.xyz/content/maxwell3_fdem/sphere_freespace/index.html

**Pull request closed without merging**